yabt/target_utils_test.py

```diff
@@ -93,7 +93,7 @@ def test_norm_name_unqualified_error():
             'possible ambiguity' in str(excinfo.value))
 
 
-_HELLO_PROG_HASH = '0e049ee8ed31c616033076f8f90415a7'
+_HELLO_PROG_HASH = '19bd1e73278f9a524de049d0d950c3af'
```

Comment:
I'm not sure why it changed. I'm guessing that since the proto builder signature have changed
But why didn't the proto traget hash changed?

Reply:
> But why didn't the proto traget hash changed?

which proto target are you referring to? the one with the hash below this (`_PROTO_BUILDER`)? it doesn't have any dependencies on proto targets, so no reason for it to change.

Reply:
great